packages/teleterm/src/ui/TopBar/NavigationMenu/NavigationMenu.tsx

```diff
@@ -59,7 +59,10 @@ export function NavigationMenu() {
   const [isPopoverOpened, setIsPopoverOpened] = useState(false);
   const selectorRef = useRef<HTMLButtonElement>();
 
-  if (!activeRootCluster) {
+  if (
+    !activeRootCluster ||
+    !activeRootCluster.features.advancedAccessWorkflows
```

Comment:
I think this is bad design. I've updated the backend to only return the `features` on the relevant rpcs (such as `GetCluster`). Although I think in the future we may want to rename `ListRootClusters` to something like `ListRootClustersInProfile` since that more clearly describes what it does

Reply:
> Although I think in the future we may want to rename `ListRootClusters` to something like `ListRootClustersInProfile` since that more clearly describes what it does

Another idea for the future would be to utilize [the render view pattern](https://cloud.google.com/apis/design/design_patterns#resource_view) that Alan mentioned. But yeah, we're free to completely change the RPCs around getting root cluster details, there's nothing holding us back. They work the way they work right now because we've never had to do more than just read stuff from the disk.